app/tag-management/all-tags/all-tags.component.css

```diff
@@ -0,0 +1,11 @@
+.page-header {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  border-style: none;
+}
+
+/deep/ .md-tab-label {
```

Comment:
What about set a class and use the class for that? Would be a cleaner solution than this if it works.